grid-demo/src/app/angular/column-ordering/column-ordering.component.ts

```diff
@@ -15,4 +16,20 @@ export class AngularColumnOrderingComponent {
   constructor() {
     this.vms = generateVms({ pageIndex: 0, pageSize: 1000 }).pipe(map(data => data.vms));
   }
+
+  columnPickedUp(columnName: string) {
+    console.log(`column \"${columnName}\" picked up`);
```

Comment:
You don't have to escape the quotes here.

Reply:
If you update this, please update the other `console.log` statements also.

Reply:
fixed